modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -100,6 +108,120 @@ You must have the required permissions to create the Kubernetes secret.
 
 To store your user credentials in Kubernetes secrets:
 
+. In a YAML file, create a secret in which to store the sensitive information, for example, `orders-api-config.yaml`:
++
+[source,text,linenums]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: _secret name_ # example: orders-api-config
+  namespace: _namespace name_ # example: default
+type: Opaque
+stringData:
+  config: |
+    {
+      "user": "_replace-with-anypoint-username_",
+      "password": "_replace-with-anypoint-clientSecret_"
+    }
+----
+. Apply the secret: 
++
+`kubectl apply -f orders-api-config.yaml`
++
+If the secret is correctly applied, a 'secret/orders-api-config created' message is displayed.
++
+. Reference the secret in your API CRD (instead of the user and password parameter), as illustrated in the following example:
++
+[source,text,linenums]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: _Kubernetes API resource name_ # example: orders-api
+  namespace: _namespace name_ # example: default
+  labels:
+    service-mesh.mulesoft.com/kind: api
+spec:
+  clusterServiceClassExternalName: anypoint-platform-api-instance
+  clusterServicePlanExternalName: service-mesh
+  parameters:
+    environmentId: _environment ID_ # example: 7cea45c6-3237-464d-bd34-3122c4abde29
+    groupId: _group ID_ # Optional - example: 350b8164-e809-4091-91f5-079fb30e5a1e
+    assetId: _asset ID_ # example: orders-api
+    version: _version_ # example: 0.0.1
+    instanceLabel: _instance label_ # example: public
+    tags:
+      apiSpec:
+      - _tag in Exchange asset_ # example: orders
+      apiInstance:
+      - _tag in API Manager instance_ # example: orders
+    clientId: _ID of the Anypoint Platform connected app_
+    clientSecret: _Secret of the Anypoint Platform connected app_
+----
+
+[[create-api-CRD-ConnAps]]
+== Create an API Using Anypoint Service Mesh With Client Credentials
+
+Use the procedure in this section to automatically create an API using Anypoint Service Mesh using your Conncted App client credentials. Before you create the API, you must complete the task prerequisite.
```

Comment:
What do you think of changing the wording for something like:
```suggestion
Use the procedure in this section to automatically create an API using Anypoint Service Mesh using the credentials of your Conncted App. Before you create the API, you must complete the task prerequisite.
```

Feel free to discard my suggestion.
Please, if you choose to apply my suggestion, apply it also in `create-an-api-configure-service-mesh-CLI.adoc`

Reply:
Done in both files:
Use the procedure in this section to automatically create an API using Anypoint Service Mesh with your Connected App credentials.
Thanks!